dali/kernels/slice/slice_flip_normalize_permute_gpu.h

```diff
@@ -102,7 +102,7 @@ class SliceFlipNormalizePermutePadGPU {
       norm_add_cpu[i] = -mean_data[i] * inv_stddev_data[i];
       norm_mul_cpu[i] = inv_stddev_data[i];
     }
-    unsigned normalization_dim;
+    unsigned normalization_dim = Dims + 1;
```

Comment:
was it a bug?

Reply:
Not really, but the compiler generated a warning and CI failed :(